packages/flutter_adaptive_scaffold/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
+## 0.1.1
+
+* 🐛 : FIX - Navigation Rail items not considering `NavigationRailTheme` values - [121135](https://github.com/flutter/flutter/issues/121135)
+* When `NavigationRailTheme` is provided, it shall that consider that if user has not given theme related data explicitly.
```

Comment:
```suggestion
* When `NavigationRailTheme` is provided, it will use the theme for values that the user has not given explicit theme-related values for.
```

Reply:
change log updated .